corehq/apps/app_manager/tests/util.py

```diff
@@ -173,15 +174,13 @@ def commtrack_enabled(is_enabled):
 
 @unit_testing_only
 def delete_all_apps():
-    results = Application.get_db().view(
-        'app_manager/applications',
-        reduce=False,
-        include_docs=False,
-    ).all()
-    for result in results:
-        try:
-            app = Application.get(result['id'])
-        except Exception:
-            pass
-        else:
-            app.delete()
+    for doc_type in app_doc_types():
+        res = Application.get_db().view(
+            'all_docs/by_doc_type',
+            startkey=[doc_type],
+            endkey=[doc_type, {}],
+            reduce=False,
+            include_docs=True
+        )
+        for row in res:
+            Application.get_db().delete_doc(row['doc'])
```

Comment:
what's the reason behind this change?
these two now seem to be different things, my understanding is that `app.delete` would call other model callbacks but now this won't.

Reply:
I'm not aware of any 'callbacks' we do on deletion of models. The reason for doing this is to avoid having to wrap the doc.